docs/ADR/0023-string-interpolation-and-binaries.md

```
@@ -0,0 +1,410 @@
+
+The `printString` message is already the standard way to get a string representation in beamtalk (following Smalltalk convention). This is analogous to:
+- Elixir's `String.Chars.to_string/1` protocol
+- Python's `__str__`
+- Ruby's `to_s`
+- Pharo's `asString`
+
+If `printString` is not understood by the receiver, the standard `doesNotUnderstand:` error is raised — no silent failures.
+
+### REPL Session
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)